examples/models/legends.py

```diff
@@ -1,3 +1,11 @@
+'''This example shows different ways to place legends on a graph using LegendLocation. 
```

Comment:
This example is not using LegendLocation to place legends. Rather, it is using the Legend object to place legends in different locations, all of which are included in the enum LegendLocation.

How about:

```suggestion
'''This example shows different ways to place legends on a graph using the different location values included in the enumeration ``LegendLocation``. 
```